Integrations/CiscoASA/CiscoASA.yml

```diff
@@ -0,0 +1,446 @@
+category: Network Security
+commonfields:
+  id: Cisco ASA
+  version: -1
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  required: true
+  type: 0
+- display: Credentials
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: is ASAv
+  name: isASAv
+  required: false
+  type: 8
+description: Manage Cisco ASA
```

Comment:
elaborate more about the capabilities here (also what ASA stands for)

Reply:
Cisco Adaptive Security Appliance Software is the core operating system for the Cisco ASA Family. It delivers enterprise-class firewall capabilities for ASA devices